dockstore-integration-testing/src/test/java/io/dockstore/client/cli/OrganizationIT.java

```diff
@@ -810,8 +811,116 @@ public void testRequestUserJoinOrgAndApprove() {
         assertNull("Other user should not be able to update the Organization.", organization);
     }
 
+    @Test
+    public void testMembersArePowerless() {
+        final ApiClient webClientUser2 = getWebClient(USER_2_USERNAME, testingPostgres);
+        OrganizationsApi organizationsApiUser2 = new OrganizationsApi(webClientUser2);
+
+        // Setup other user
+        final ApiClient webClientOtherUser = getWebClient(OTHER_USERNAME, testingPostgres);
+        OrganizationsApi organizationsApiOtherUser = new OrganizationsApi(webClientOtherUser);
+        UsersApi usersOtherUser = new UsersApi(webClientOtherUser);
+
+        // Create org, invite user as member, and accept invitation
+        Organization organization = createOrg(organizationsApiUser2);
+        long orgId = organization.getId();
+        long userId = 1;
+        long otherUserId = 2;
+        organizationsApiUser2.addUserToOrg(OrganizationUser.Role.MEMBER.toString(), otherUserId, orgId, "");
+        organizationsApiOtherUser.acceptOrRejectInvitation(orgId, true);
+
+        // Should not be able to update organization information
+        String email = "hello@email.com";
+        organization.setEmail(email);
+        try {
+            organization = organizationsApiOtherUser.updateOrganization(organization, orgId);
+            Assert.fail("Should not be able to update organization information");
+        } catch (ApiException ex) {
+            Assert.assertEquals(HttpStatus.SC_UNAUTHORIZED, ex.getCode());
+        }
+
+        // Should not be able to invite another user
+        long thirdUser = 3;
+        try {
+            organizationsApiOtherUser.addUserToOrg(OrganizationUser.Role.MEMBER.toString(), thirdUser, orgId, "");
+            Assert.fail("Member should not be able to add a user to organization");
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'Assert.fail' due to existing static import 'org.junit.Assert.fail'](https://app.codacy.com/gh/dockstore/dockstore/pullRequest?prid=6348842)